source/common/stats/tag_extractor_impl.h

```diff
@@ -50,8 +52,31 @@ class TagExtractorImpl : public TagExtractor {
   const std::string name_;
   const std::string prefix_;
   const std::string substr_;
+};
+
+class TagExtractorStdRegexImpl : public TagExtractorImplBase {
+public:
+  TagExtractorStdRegexImpl(const std::string& name, const std::string regex,
```

Comment:
nit: `regex` should be a const reference. Same below.

Reply:
actually I switched most of these to absl::string_view.

Reply:
Even better, thanks!